### DISCUSSION TITLE
How to lookup which lotus client initiated the transfer transaction？

### DISCUSSION THREAD

**Discussion by Tangqiaoyehua** [filecoin-project/lotus#8054] [General Q&A]
A lotus client initiated one transfer transaction with command :"lotus send --from **"  in my lotus cluster . I have no idea which client initiated this transaction. 
So I wonder to know  how to lookup which lotus client initiated the transfer transaction？ Or  how to get some useful info ? like ip address,client id .

**Comment by LexLuthr** [comment]
Can you explain it bit more? Do you have multiple lotus lite nodes attached to a lotus node? What do you mean by a cluster?
You can see the wallet id of the transaction the message.

**Comment by LexLuthr** [comment]
I am archiving this discussion as there has been no new input from the author for more than 7 days.
